app/events/event_types/inventory.rb

```diff
@@ -30,6 +30,14 @@ def set_item_quantity(item_id:, quantity:, location:)
     # @param to_location [Integer]
     # @param validate [Boolean]
     def move_item(item_id:, quantity:, from_location: nil, to_location: nil, validate: true)
+      if quantity.negative?
+        move_item(item_id: item_id,
+          quantity: -quantity,
+          from_location: to_location,
+          to_location: from_location,
+          validate: validate)
+        return
```

Comment:
Subtle suggestion:

```suggestion
        return move_item(item_id: item_id,
          quantity: -quantity,
          from_location: to_location,
          to_location: from_location,
          validate: validate)
```

This looks good, but I'll have to go back to understand when this happens operationally